### PR TITLE
Remove link from bottom corner for jp

### PR DESCRIPTION
**Pull request by 4p00rv**

### CODE REVIEW

File: src/charts/charts.js [hunk 1]
```diff
@@ -263,12 +263,15 @@ define(["jquery","charts/chartingRequestMap", "websockets/binary_websockets", "w
                             if ($.isFunction(onload)) {
                                 onload();
                             }
-
-                            this.credits.element.onclick = function() {
-                                window.open(
-                                    'http://webtrader.binary.com',
-                                    '_blank'
-                                );
+                            if(getParameterByName("affiliates") === 'true' && getParameterByName('lang') === 'jp'){
```
Comment (arnabk) [inline src/charts/charts.js]: @4p00rv What if someone sends JP?

File: src/images/chart_template-w.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50"><defs><style>.cls-1{fill:#fff;}</style></defs><title>chart_template</title><rect class="cls-1" x="9" y="16" width="6" height="25"/><rect class="cls-1" x="22" y="21" width="6" height="20"/><rect class="cls-1" x="35" y="9" width="6" height="32"/></svg>
```
Comment (arnabk) [inline src/images/chart_template-w.svg]: @4p00rv Whats this for?

Comment (4p00rv) [inline src/images/chart_template-w.svg]: Git added changes to my other branch to this one. Sorry for that. reverting.

**Comment by arnabk** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/binary-com/webtrader/pull-request/502/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=binary-com/webtrader)

**Pull request merged**